### PR TITLE
yapf on deepchem/utils part two

### PR DESCRIPTION
**Pull request by maithili232**
# Pull Request Template

## Description

Fix #3159 

## Type of change

Please check the option that is related to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
- [x] Documentations (modification for documents)

## Checklist

- [x] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
  - [x] Run `yapf -i <modified file>` and check no errors (**yapf version must be  0.32.0**)
  - [ ] Run `mypy -p deepchem` and check no errors
  - [ ] Run `flake8 <modified file> --count` and check no errors
  - [ ] Run `python -m doctest <modified file>` and check no errors
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: deepchem/utils/evaluate.py [hunk 1]
```diff
@@ -18,479 +18,485 @@
...
     Parameters
     ----------
     model: Model
-      Model to evaluate.
-    generator: generator
-      Generator which yields batches to feed into the model. For a
-      KerasModel, it should be a tuple of the form (inputs, labels,
-      weights). The "correct" way to create this generator is to use
-      `model.default_generator` as shown in the example above.
+      Model to evaluate. Note that this must be a regression or
```
Comment (gusty1g) [inline deepchem/utils/evaluate.py:185]: Formatting of docstrings should be performed in these lines.

**Review by gusty1g — CHANGES_REQUESTED** [review]
At some parts, docstring should be formatted. Also, the checklist in the pull request template must be filled up.

File: deepchem/utils/fake_data_generator.py [hunk 2]
```diff
@@ -8,155 +8,158 @@
...
-  edge_index = np.random.randint(low=0, high=n_nodes, size=(2, n_edges))
-
-  if remove_loops:
-    edge_index = remove_self_loops(edge_index)
-  return edge_index
+    """Returns source and destination nodes for `num_nodes * avg_degree` number of randomly
+    generated edges. If remove_loops is True, then self-loops from the edge_index pairs
+    are removed.
+
+    Parameters
```
Comment (advikavs) [inline deepchem/utils/fake_data_generator.py:135]: 135-143 needs to be formatted

**Comment by gusty1g** [comment]
doctest is failing in Unit Test CI for Electron Sampler.

**Review by gusty1g — APPROVED** [review]

**Pull request merged**